caffe2/python/convnet_benchmarks.py

```diff
@@ -697,10 +776,11 @@ def GetArgumentParser():
         action='store_true',
         help="If True, dump the model prototxts to disk."
     )
-    parser.add_argument("--net_type", type=str, default="dag")
+    parser.add_argument("--net_type", type=str, default="simple")
```

Comment:
Any specific reason to run the Resnet model as simple net rather than default DAG?

Reply:
I remember that we were facing some problems with dag net when layerwise_benchmarks was on. Let me check if that's not a problem anymore.